components/kubearchive/base/otel-collector-config.yaml

```diff
@@ -13,10 +11,17 @@ processors:
 exporters:
   prometheus:
     endpoint: '0.0.0.0:9090'
+    send_timestamps: true
+    add_metric_suffixes: false
+  debug:
```

Comment:
is this correct @rh-hemartin ? I mean to have the `debug`section defined and nothing under?

Reply:
Yes, its defining the `debug` exporter where the traces are sent to.